tappsbt/mock.go

```diff
@@ -338,6 +338,29 @@ func NewTestFromVInput(t testing.TB, i *VInput) *TestVInput {
 		ti.Proof = proof.NewTestFromProof(t, i.Proof)
 	}
 
+	if len(i.AltLeaves) > 0 {
+		// Assert that the concrete type of AltLeaf is supported.
+		switch i.AltLeaves[0].(type) {
```

Comment:
Too bad the property testing framework doesn't play nice with `require`. Otherwise this could just be `require.IsType(t, &asset.Asset{}, v.AltLeaves[0])`.

Reply:
Ah didn't know about that in `require`!

FWIW I think usage of `require` can be replaced with `assert` + `Errorf` in most cases.